OSDOCS-16993:CQA for Azure install (#114095)
openshift/openshift-docs: installing/installing_azure/installing-azure-account.adoc

:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="installing-azure-account"]
= Configuring an {azure-short} account
:context: installing-azure-account

toc::[]

[role="_abstract"]

> To ensure {product-title} can successfully install and run the cluster on Microsoft Azure, you must configure an {azure-short} account with the correct identity and permissions before you start the installation.

Reviewer: IMO I kind of liked how the previous one at least started, maybe you could emulate that for brevity?

```suggestion
Before you can install {product-title} on {azure-first}, you must configure an {azure-short} account with the correct identity and permissions before you start the installation.
```

[IMPORTANT]
====

> All {azure-short} resources that are available through public endpoints are subject to resource name restrictions. For a list of terms that {azure-short} restricts for resource names, see the {azure-short} documentation linked in _Additional resources_.

Reviewer: In general I think when removing links it's still valuable to keep the destination name in quotes, as it more clearly hints to users what to look for:

```suggestion
All {azure-short} resources that are available through public endpoints are subject to resource name restrictions. For a list of terms that {azure-short} restricts for resource names, see "Resolve errors for reserved resource names".
```
(I updated the name to match what the heading is in the actual destination)

====

include::modules/installation-azure-limits.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

> * link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-reserved-resource-name[Resolve reserved resource name errors in the {azure-short} documentation]

Reviewer: Two things

1. I think this way would more closely replicate how the OSDOCS guidelines suggest we indicate external links:
```suggestion
* link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-reserved-resource-name[Resolve errors for reserved resource names ({azure-short} documentation)]
```

2. When it comes to removing links from assembly intros, I think finding the right additional resources section is hard. Putting it under the next module's AR section is technically the closest you can get the link to its original location, but on the other hand, putting in an assembly level AR section at least gives users a heading to find in the right-side TOC.

In either case, if you reference the exact link name in the admonition above a user can ctrl + F it, so no big deal, but I would maybe consider moving this to an assembly level AR section instead

Author: Moved to AR in TOC.

include::modules/installation-azure-regions.adoc[leveloffset=+1]

[role="_additional-resources"]

> .Additional resources

Reviewer: I think this should be assembly level, if the next steps was an assembly level heading:

```suggestion
[id="additional-resources_{context}"]
== Additional resources
```

Author: Sorry, got too heavy handed with my .Additional resources changes 😄
